controllers/cluster_controller.go

```diff
@@ -114,8 +114,9 @@ func (r *FoundationDBClusterReconciler) Reconcile(request ctrl.Request) (ctrl.Re
 		UpdateLabels{},
 		UpdateDatabaseConfiguration{},
 		ChooseRemovals{},
-		ChangeCoordinators{},
 		ExcludeInstances{},
+		ChangeCoordinators{},
+		ConfirmExclusionCompletion{},
 		BounceProcesses{},
```

Comment:
Is this safe to do if we're excluding a coordinator?

Reply:
Yes, it should be safe to start the exclusion while the process is still a coordinator. We just need to make sure we change coordinators before removing the process, which this sequence will accomplish.